### PR TITLE
[office-js, office-js-preview] Adding descriptions to async overloads

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.microsoft.com/en-us/javascript/api/overview/office?view=office-js

### CODE REVIEW

**Comment by AlexJerabek** [comment]
@Zlatkovsky, you had requested this recently. Just making sure your eyes cross the update.

**Comment by PranavSenthilnathan** [comment]
I'm assuming these are all auto-generated? Do you want to wait for reviewers or can I go ahead and merge this in?

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -3962,18 +5169,16 @@ declare namespace Office {
          * 
          * *Supported hosts, by platform*
          *  <table>
-         *   <tr><th>                          </th><th> Office for Windows desktop </th><th> Office Online (in browser)</th></tr>
-         *   <tr><td><strong> Project </strong></td><td> Y                          </td><td>                           </td></tr>
+         *   <tr><th>                    </th><th> Office for Windows desktop </th><th> Office Online (in browser)</th></tr>
+         *   <tr><td><strong> Project    </strong></td><td> Y                          </td><td>                           </td></tr>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts]: nit: this had been spaced so that the first closing th and td lined up (vs the strong and the th)

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Have one small thing I noticed but it's not blocking.

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Fixed and ready to check in.

**Pull request merged**